### ISSUE TITLE
SSH Session Killed : Improvements at Server Keep

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#1020]
# Feature description
Some times SSH sessions are killed because app goes to background for too long or because we are on the go and we loose the carrier.

It should be powerful to maintain the connection on the server itself and when we reconnect we can attach the session again without loosing anything.

**Comment by nothingto** [comment]
This can possibly be something solved through mosh no?

**Comment by isontheline** [comment]
@nothingto Yes but ad mosh license isn’t compatible with WebSSH I need to rewrite it or find a lighter solution
